src/lib/integration_test_cloud_engine/graphql_polling_log_engine.ml

```diff
@@ -56,7 +56,7 @@ let parse_event_from_log_entry ~logger log_entry =
       in
       event )
 
-let rec poll_get_filtered_log_entries_node ~logger ~event_writer
+let rec get_filtered_log_entries_of_node ~logger ~event_writer
```

Comment:
Isn't `get` usually used when there's a single definitive result? It's probably clearer to make reference to `poll` or `stream` and drop the `get`.

Reply:
every function beginning with the word `poll_` was confusing me, and i wanted to make the names more distinct at a glance without having to fully read the whole thing.  as for this function, even though it's looping, it's still `get`ing a relatively definitive thing which is the log entries (or at least that's how I figured)

Reply:
ok nm i changed it anyways